examples/cursor_grab.rs

```diff
@@ -36,6 +36,7 @@ fn main() {
                         Escape => *control_flow = ControlFlow::Exit,
                         G => window.set_cursor_grab(!modifiers.shift()).unwrap(),
                         H => window.set_cursor_visible(modifiers.shift()),
+                        I => window.set_ignore_cursor_events(!modifiers.shift()).unwrap(),
```

Comment:
I'm not sure if this quite fits in with the `cursor_grab` example, but that's mostly a naming thing for me.